docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -56,30 +63,24 @@ fun MessageListHeader(
 )
 ```
 
-* `onBackPressed`: Handler for the `BackButton` click action.
+* `onBackPressed`: Handler for the header `BackButton` click action.
 * `onHeaderActionClick`: Handler when the user clicks on the title for more information about the channel.
 
+:::note
+The `leadingContent` uses the `onBackPressed` parameter as its default action, while the `centerContent` uses the `onHeaderActionClick` parameter as its default action. If you want to keep the same UI but change the behavior, you need to override `onBackPressed` or `onHeaderActionClick`. Otherwise, skip over to [Customization](#customization) to learn how to customize the UI.
```

Comment:
The `leadingContent` uses `onBackPressed` as its default action, while the `centerContent` uses `onHeaderActionClick`. If you want to keep the same UI but change the behavior, you can override `onBackPressed` or `onHeaderActionClick`. 

Otherwise, skip over to [Customization](#customization) to learn how to customize the UI.